cvat/apps/dataset_manager/views.py

```diff
@@ -85,7 +85,10 @@ def export(dst_format, project_id=None, task_id=None, job_id=None, server_url=No
                 os.replace(temp_file, output_path)
 
             archive_ctime = osp.getctime(output_path)
-            scheduler = django_rq.get_scheduler(settings.CVAT_QUEUES.EXPORT_DATA.value)
+            scheduler = django_rq.get_scheduler(
+                settings.CVAT_QUEUES.EXPORT_DATA.value,
+                interval=settings.DJANGO_RQ_SCHEDULER_INTERVAL
```

Comment:
Are you sure the interval should be set here and not in the scheduler startup parameters?